aten/src/ATen/native/mkldnn/xpu/qconv_pt2e.cpp

```diff
@@ -155,8 +156,15 @@ class QConvoneDNNXPU final {
         stride.vec(),
         dilation.vec());
 
+    bool fp32_output =
+        output_dtype.has_value() && (output_dtype == c10::kFloat);
+    bool bfloat16_output =
+        output_dtype.has_value() && (output_dtype == c10::kBFloat16);
+    auto dst_dtype = fp32_output
+        ? c10::kFloat
+        : (bfloat16_output ? c10::kBFloat16 : c10::kByte);
```

Comment:
What is the logic of this part?  can you add more comments about it? how about fp16?